docs/account-management/users/create-manage-users.md

```diff
@@ -0,0 +1,37 @@
+---
+sidebar_position: 20
+title: Create and Manage Users
+---
+
+This section describes how to create and manage user accounts.
+
+## Create User
+
+Follow these instructions to create a new user using the GDN console web UI.
+
+1. [Log in to your Macrometa account](https://auth.paas.macrometa.io/).
+1. Click **Account**.
+1. Click **New User**.
+1. Enter a unique username.
+1. Enter the email to be associated with this user account.
+1. Enter a password and confirm it.
+1. Optionally, choose to create the account as inactive by deselecting **Active**.
+
+## Change Password
+
+On the Users tab, you can select a user to access their details and change their password.
+
+1. [Log in to your Macrometa account](https://auth.paas.macrometa.io/).
+1. Click **Account**.
+1. Select the user whose password you want to change.
+1. Click **Edit** in the **Password** field.
+1. Enter and confirm the new password, then click **Save**.
+
+## Delete User
+
+On the Users tab, you can select a user to access their details and delete the account.
+
+1. [Log in to your Macrometa account](https://auth.paas.macrometa.io/).
+1. Click **Account**.
+1. Select the user you want to delete.
```

Comment:
```suggestion
1. Select the user that you want to delete.
```

Reply:
While technically not necessary, signposts like "that" help non-native English readers understand the prose.